// Copyright (c) 2021 The Orbit Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package orbit_client_protos;

message FrameTracksInfo {
  reserved 1;
  repeated uint64 frame_track_function_ids = 2;
}

message UserDefinedCaptureInfo {
  FrameTracksInfo frame_tracks_info = 1;
}